my_list.rb

```diff
@@ -0,0 +1,31 @@
+require './my_enumerable'
```

Comment:
- [ ]  it would be better to use `require_relative` because it's the one that is used to import something in your directory while `require` used for import external files like gems this also a convention in the ruby community so I encourage you to do that